doc/interesting-queries.md

```diff
@@ -298,6 +298,82 @@ from the treasury. These can be coalesced into single query via an SQL `union` o
 ...
 ```
 
+### Get historical UTxO set for a given timestamp
+The UTxO set is dependent on time, this will return it for a given timestamp
+```
+WITH const AS (
+  SELECT TO_TIMESTAMP('2020-10-10 17:00:00', 'YYYY-MM-DD HH24:MI:SS') AS effective_time_
+)
+
+# SELECT tx_out.address        AS address
+    ,    (tx_out.value / 1e6)  AS ada
+    ,    generating_block.time AS timestamp
+
+  FROM const
+  CROSS JOIN tx_out
+  JOIN       tx     AS generating_tx    ON generating_tx.id             = tx_out.tx_id
```

Comment:
Best not to vertically align anything other than indentation. As soon as one of these names change, the alignment wil be off. You should try to match the way I have done it elsewhere in this file.